### PR TITLE
merge downstream changes for build config, signup and import

### PR DESCRIPTION
**Pull request by sbtaylor15**

### CODE REVIEW

File: chart/ms-ui/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        track: stable
+    spec:
+      containers:
+        - name: dh-ms-ui
+          image: "{{ .Values.DockerRepo }}:{{ .Values.DockerTag }}"
+          imagePullPolicy: Always
+          ports:
+            - name: http
+              containerPort: 8080
+---
```
Comment (markjacksonfishing) [inline chart/ms-ui/templates/deployment.yaml]: Non-blocking nit: Can we add a line feed

File: chart/ms-ui/templates/service.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  selector:
+    name: {{ include "ms-ui.name" . }}
+    tier: backend
+  ports:
+    - protocol: TCP
+      port: 8080
+      nodePort: 30091
+  type: NodePort
+---
+
```
Comment (markjacksonfishing) [inline chart/ms-ui/templates/service.yaml]: Non-blocking nit: Can we add a line feed

File: cloudbuild/cloudbuild-general.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      - 'DOCKER_CONFIG=/workspace/docker-config'
+
+secrets:
+- kmsKeyName: projects/eighth-physics-169321/locations/global/keyRings/cli/cryptoKeys/quay
+  secretEnv:
+    QUAY_USERID: CiQAW+P1J9UZz+Hr1uonladAW2dKqaiVd5ux8Q9EV81pK0u5V+4SNACcBdnKacvH4QXPamH1N4uJZvZ/0TMwvELgXAAlP0wR2zBw2WhCV82GMiUkW3iGVlbqz7c=
+- kmsKeyName: projects/eighth-physics-169321/locations/global/keyRings/cli/cryptoKeys/quay-pw
+  secretEnv:
+    QUAY_PASSWORD: CiQAUULEud9Ej8XtwNAb9gkbDVhSGFZYhUGE30fNwR+7ehAOkH8SMgCz6KYeykjgS16RPxgKlrIQL/1TKDt06v4OXGIisFXOkdWC+jvdda8mTzVNCi8sT5g6 
+
```
Comment (markjacksonfishing) [inline cloudbuild/cloudbuild-general.yaml]: Non-blocking nit: Can we add a line feed

File: cloudbuild/cloudbuild-nginx.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      - 'DOCKER_CONFIG=/workspace/docker-config'
+
+secrets:
+- kmsKeyName: projects/eighth-physics-169321/locations/global/keyRings/cli/cryptoKeys/quay
+  secretEnv:
+    QUAY_USERID: CiQAW+P1J9UZz+Hr1uonladAW2dKqaiVd5ux8Q9EV81pK0u5V+4SNACcBdnKacvH4QXPamH1N4uJZvZ/0TMwvELgXAAlP0wR2zBw2WhCV82GMiUkW3iGVlbqz7c=
+- kmsKeyName: projects/eighth-physics-169321/locations/global/keyRings/cli/cryptoKeys/quay-pw
+  secretEnv:
+    QUAY_PASSWORD: CiQAUULEud9Ej8XtwNAb9gkbDVhSGFZYhUGE30fNwR+7ehAOkH8SMgCz6KYeykjgS16RPxgKlrIQL/1TKDt06v4OXGIisFXOkdWC+jvdda8mTzVNCi8sT5g6 
+
```
Comment (markjacksonfishing) [inline cloudbuild/cloudbuild-nginx.yaml]: Non-blocking nit: Can we add a line feed

File: cloudbuild/cloudbuild-ui.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+      - 'DOCKER_CONFIG=/workspace/docker-config'
+
+secrets:
+- kmsKeyName: projects/eighth-physics-169321/locations/global/keyRings/cli/cryptoKeys/quay
+  secretEnv:
+    QUAY_USERID: CiQAW+P1J9UZz+Hr1uonladAW2dKqaiVd5ux8Q9EV81pK0u5V+4SNACcBdnKacvH4QXPamH1N4uJZvZ/0TMwvELgXAAlP0wR2zBw2WhCV82GMiUkW3iGVlbqz7c=
+- kmsKeyName: projects/eighth-physics-169321/locations/global/keyRings/cli/cryptoKeys/quay-pw
+  secretEnv:
+    QUAY_PASSWORD: CiQAUULEud9Ej8XtwNAb9gkbDVhSGFZYhUGE30fNwR+7ehAOkH8SMgCz6KYeykjgS16RPxgKlrIQL/1TKDt06v4OXGIisFXOkdWC+jvdda8mTzVNCi8sT5g6 
+
```
Comment (markjacksonfishing) [inline cloudbuild/cloudbuild-ui.yaml]: Non-blocking nit: Can we add a line feed

File: cloudbuild/cloudbuild.toml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+[Component]
+    Components = [
+         { Name = "GLOBAL.SF Tech.Online Store Company.Purchase Processing.Cart Service.cartservice", Application = "GLOBAL.SF Tech.Online Store Company.Hipster Store.Dev.Hipster Store;Labor Day Sale;1_2_10_2"}
+    ]    
+    Version = "1.2.2"
+    ChartName = "chart/cartservice"
+    DockerRepo = "quay.io/hipsterstore/cartservice"
+    GitUrl = "https://github.com/ortelius/store-cartservice"
+    CustomAction = "GLOBAL.HelmChart"
```
Comment (markjacksonfishing) [inline cloudbuild/cloudbuild.toml]: Non-blocking nit: Can we add a line feed

File: cloudbuild/cloudbuild.yaml [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+      - 'DOCKER_CONFIG=/workspace/docker-config'
+
+secrets:
+- kmsKeyName: projects/eighth-physics-169321/locations/global/keyRings/cli/cryptoKeys/quay
+  secretEnv:
+    QUAY_USERID: CiQAW+P1J9UZz+Hr1uonladAW2dKqaiVd5ux8Q9EV81pK0u5V+4SNACcBdnKacvH4QXPamH1N4uJZvZ/0TMwvELgXAAlP0wR2zBw2WhCV82GMiUkW3iGVlbqz7c=
+- kmsKeyName: projects/eighth-physics-169321/locations/global/keyRings/cli/cryptoKeys/quay-pw
+  secretEnv:
+    QUAY_PASSWORD: CiQAUULEud9Ej8XtwNAb9gkbDVhSGFZYhUGE30fNwR+7ehAOkH8SMgCz6KYeykjgS16RPxgKlrIQL/1TKDt06v4OXGIisFXOkdWC+jvdda8mTzVNCi8sT5g6 
+
```
Comment (markjacksonfishing) [inline cloudbuild/cloudbuild.yaml]: Non-blocking nit: Can we add a line feed

File: docker/Dockerfile [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+WORKDIR /build/
+
+COPY . /workspace
+RUN /workspace/tgt/build.sh
+
+FROM quay.io/ortelius/ortelius-base:latest
+
+WORKDIR /opt/deployhub/
+COPY --from=0 /workspace/installers/linux/webadmin/webapp-runner.jar webadmin/webapp-runner.jar
```
Comment (markjacksonfishing) [inline docker/Dockerfile]: COPY --from should reference a previously defined FROM alias?

File: docker/Dockerfile [hunk 9]
```diff
@@ -0,0 +1,26 @@
+FROM quay.io/openmake/meister:latest
```
Comment (markjacksonfishing) [inline docker/Dockerfile]: Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

File: docker/entrypoint.sh [hunk 10]
```diff
@@ -1,27 +1,30 @@
 #!/bin/sh
```
Comment (markjacksonfishing) [inline docker/entrypoint.sh:1]: On line 40:
^The mentioned syntax error was in this if expression.
    ^Couldn't parse this test expression. Fix to allow more checks.
         Expected this to be an argument to the unary condition.
                                ^You need a space before the ].
                                ^Missing space before ]. Fix any mentioned problems and try again.

**Review by markjacksonfishing — CHANGES_REQUESTED** [review]
See inline comments

**Comment by sbtaylor15** [comment]
Made changes per comments.  Please review again.

**Review by markjacksonfishing — APPROVED** [review]
/lgtm

**Comment by sbtaylor15** [comment]
Missed one FROM alias.

**Review by markjacksonfishing — APPROVED** [review]

**Comment by sbtaylor15** [comment]
Merging downstream maintenance changes, cloudbuild and dockerfile updates, API import updates and saas signup

**Pull request merged**